### DISCUSSION TITLE
Pylance issue

### DISCUSSION THREAD

**Discussion by michaelxswe** [devcontainers/community#167] [Q&A]
devcontainer.json
```
{
    "name": "app",
    "dockerComposeFile": "../docker-compose.yml",
    "service": "app",
    "workspaceFolder": "/workspace",
	"customizations": {
    	"vscode": {
      	    "extensions": [
	        "ms-python.python",
		"charliermarsh.ruff"
	    ]
    	}
    }
}
```

Whenever I try to create a new file, pylance won't recognize the new file and cause import to fail

For example if I create a new file and try to import it from main.py, i will get the following error

```
Import "app.sub" could not be resolvedPylance[reportMissingImports]
```

Now if I reload the the ide window, import works as expected.

Does anyone know how I can resolve this?

**Comment by bamurtaugh** [comment]
Thanks for sharing this. I wonder if @luabud might have any thoughts?

**Comment by luabud** [comment]
I'm actually not sure what could be happening! Would you mind filing a bug in Pylance by following  the template in https://github.com/microsoft/pylance-release/issues/new?template=bug-report.md? It'd be super helpful if you could share contents of the "Python Language Server" Output window (View > Command Palette... and run "Pylance: go to output channel...") so our team is able to investigate better.

**Comment by michaelxiacs** [comment]
Seems to be resolved by the latest update
